android/src/app/organicmaps/car/NavigationCarAppService.java

```diff
@@ -25,6 +44,53 @@ public HostValidator createHostValidator()
   @Override
   public Session onCreateSession()
   {
-    return new NavigationSession();
+    createNotificationChannel();
+
+    // Turn the car app service into a foreground service in order to make sure we can use all
```

Comment:
Yes, point 2 is exactly what I'm planning to do in the future. Not in this PR because it's already big enough.

AA must have its foreground service because it's not working correctly without it.
For the case of this PR:
* We don't have foreground service for AA
* We close AA
* Application is fully destroyed and recreated
* It's unable to detect disconnecting of AA

That's why I added this dummy foreground service right now